obspy/core/inventory/response.py

```diff
@@ -1551,7 +1551,7 @@ def approximation_type(self, value):
         self._approximation_type = value
 
 
-class FilterCoefficient(CustomFloat):
+class FilterCoefficient(FloatWithUncertainties):
```

Comment:
I don't think so - the `FilterCoefficient` class is used for the coefficients of the FIR filter stage which takes doubles with no errors. Maybe we can just make this a simple list and infer the number argument from the position in the list? This would of course only work if they always start at 0 or 1 and always increase by one.

![screen shot 2016-07-11 at 13 25 27](https://cloud.githubusercontent.com/assets/800487/16729202/30a436be-476b-11e6-83f2-1c4658065923.png)